src/helpers/getSolanaRPCUrl.ts

```diff
@@ -0,0 +1,12 @@
+import { web3 } from '@coral-xyz/anchor';
+
+import { STATUS } from '@/constants/enum.js';
+import { env } from '@/constants/env.js';
+
+export function getSolanaRPCUrl() {
+    const envUrl = env.external.NEXT_PUBLIC_SOLANA_RPC_URL;
+
+    return env.external.NEXT_PUBLIC_SOLANA_DEV === STATUS.Enabled
+        ? web3.clusterApiUrl('devnet')
+        : envUrl || web3.clusterApiUrl('mainnet-beta');
```

Comment:
```suggestion
```suggestion
    return env.external.NEXT_PUBLIC_SOLANA_DEV === STATUS.Enabled
        ? web3.clusterApiUrl('devnet')
        : env.external.NEXT_PUBLIC_SOLANA_RPC_URL || web3.clusterApiUrl('mainnet-beta');
```
```